image/checkpoint-pod.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: kube-api-checkpoint
+  namespace: kube-system
+spec:
+  containers:
+  - name: checkpoint
+    image: quay.io/derek_parker/bootkube:v0.1.0
```

Comment:
Hmm seems to be a bit of chicken / egg here. Makes me think we might need to build the checkpoint images separately :( -- updates will be difficult otherwise. Can't update this until an image is cut, but can't cut a new image unless tagged / release.